BooruSharp.Others/Pixiv.cs

```diff
@@ -42,6 +42,7 @@ public Pixiv()
         /// <exception cref="ArgumentNullException"/>
         /// <exception cref="AuthentificationInvalid"/>
         /// <exception cref="HttpRequestException"/>
+        [Obsolete("Login with username/password does no longer works, consider using LoginWithTokenAsync instead", error: true)]
```

Comment:
You mention `LoginWithTokenAsync` method that you later removed in d503638ba20624e3747e85da0e1ed3c32d08a489.
```suggestion
        [Obsolete("Login with username/password no longer works, consider using LoginAsync with refresh token instead", error: true)]
```